tests/Validation/ValidationValidatorTest.php

```diff
@@ -1787,15 +1787,13 @@ public function testValidateMax()
         $this->assertFalse($v->passes());
 
         $file = $this->getMockBuilder(UploadedFile::class)->setMethods(['isValid', 'getSize'])->setConstructorArgs([__FILE__, basename(__FILE__)])->getMock();
-        $file->expects($this->any())->method('isValid')->willReturn(true);
-        $file->expects($this->at(1))->method('getSize')->willReturn(3072);
-        $v = new Validator($trans, ['photo' => $file], ['photo' => 'Max:10']);
+        $file->method('isValid')->willReturn(true);
+        $file->method('getSize')->willReturn(4096);
+
+        $v = new Validator($trans, ['photo' => $file], ['photo' => 'Max:4']);
         $this->assertTrue($v->passes());
 
-        $file = $this->getMockBuilder(UploadedFile::class)->setMethods(['isValid', 'getSize'])->setConstructorArgs([__FILE__, basename(__FILE__)])->getMock();
-        $file->expects($this->at(0))->method('isValid')->willReturn(true);
-        $file->expects($this->at(1))->method('getSize')->willReturn(4072);
-        $v = new Validator($trans, ['photo' => $file], ['photo' => 'Max:2']);
+        $v = new Validator($trans, ['photo' => $file], ['photo' => 'Max:3']);
```

Comment:
Can you please revert the changes to the behavior of the tests here? Just focus on the `at` method.

Reply:
Sorry. It seemed like it could be tested with one file, so I did this. I'll put it back